docker/django/Dockerfile

```diff
@@ -2,34 +2,60 @@ ARG PYTHON_VERSION=3.9
 ARG IMAGE_VARIANT=slim
 
 # ==============================
-FROM helsinkitest/python:${PYTHON_VERSION}-${IMAGE_VARIANT} AS base_stage
+FROM helsinkitest/python:${PYTHON_VERSION}-${IMAGE_VARIANT} AS appbase
 # ==============================
+RUN mkdir /entrypoint
 
-
-ENV PYTHONDONTWRITEBYTECODE true
-ENV PYTHONUNBUFFERED true
+ENV PYTHONDONTWRITEBYTECODE=true
+ENV PYTHONUNBUFFERED=true
```

Comment:
Does this work? These are usually 1s

Reply:
Afaik they should work, documentation says "if this is set to a non-empty string" for both.

Reply:
Could've googled this myself before asking. Yes, it does. https://docs.python.org/3/using/cmdline.html#envvar-PYTHONDONTWRITEBYTECODE